addons/mail/models/res_partner.py

```diff
@@ -20,6 +22,15 @@ class Partner(models.Model):
     vat = fields.Char(tracking=5)
     # channels
     channel_ids = fields.Many2many('mail.channel', 'mail_channel_member', 'partner_id', 'channel_id', string='Channels', copy=False)
+    # add tracked field
```

Comment:
Not very explicit, would maybe mention that it's not used anywhere but we value the tracking for it because blabla.
(Will help in case someone tries to grep and sees it's not used).